src/main/joyride/sci.cljs

```diff
@@ -4,7 +4,7 @@
             ["vscode" :as vscode]
             [clojure.string :as str]
             [joyride.db :as db]
-            [joyride.config :refer [workspace-scripts-path]]
+            [joyride.config :refer [workspace-abs-scripts-path user-abs-scripts-path]]
```

Comment:
I'd consider aliasing as `conf` and use these from the namespace. We do that in some other places and I kinda like to see where these functions live. (Yes, I realize the prior art here pointed in this direction instead. 😄)

Reply:
updated